src/background.js

```diff
@@ -89,15 +91,23 @@ const settings = (() => {
   };
 })();
 
-/// return channel name if it should contain a chat
+const urlRegexps = [
+  /^https?:\/\/(?:www\.)?twitch\.tv\/(\w+)\/?(?:\?.*)?$/,
+  /^https?:\/\/(?:www\.)?twitch\.tv\/popout\/(\w+)\/chat\/?(?:\?.*)?$/,
+  /^moz-extension:\/\/[a-zA-Z0-9-]+\/player\.html\?channel=(\w+)$/,
```

Comment:
This matches all extensions that have a `/player.html`. If anything, this should match specific IDs.